### ISSUE TITLE
Ability to ignore "I" of Interfaces

### ISSUE THREAD

**Issue by Akxe** [streetsidesoftware/vscode-spell-checker#292]
I have a lot of interfaces, and I do fallow the `Iname` notation. Thus I do get quite a few errors.

Is there a way to disable it, it should also probably be disabled for TS by default.

ex.: `Iplayer`, `Iads` etc

**Comment by Jason3S** [comment]
Yes, that can be very annoying.

I do not have a perfect solution. I have plans on changing the word splitter logic to handle cases like this.

For now, you can add the following to the settings files:
```
"cSpell.ignoreRegExpList": ["/\\bI(?=[a-z]+)/g"]
```

This will ignore all `I`s at the beginning of words. But it will have false positives like: `Income` will be considered incorrect.

**Comment by Jason3S** [comment]
It is also possible to do it only on a per programming language basis:

```
    "cSpell.languageSettings": [
        { "languageId": "typescript, typescriptreact", "ignoreRegExpList": ["/\\bI(?=[a-z]+)/g"] }
    ]
```

**Comment by Akxe** [comment]
Amazing, thanks

**Comment by mohamedanees6** [comment]
> I have a lot of interfaces, and I do fallow the `Iname` notation. Thus I do get quite a few errors.
> 
> Is there a way to disable it, it should also probably be disabled for TS by default.
> 
> ex.: `Iplayer`, `Iads` etc

Ideally, this must be following convention of `IName`. For Example, IPlayer, IAds etc. 

I can see that as a valid use-case.

**Comment by Akxe** [comment]
Move to convention, I originally understood it wrong
